templates/project_list.md

```diff
@@ -0,0 +1,101 @@
+## Blockchain currencies and assets 
+1.  *[Bitcoin](/projects/bitcoin.md)*des
```

Comment:
it appears so, that before taking project from project_list you need to check issues, to be sure it's not have been taken yet